Detectors/ITSMFT/ITS/tracking/include/ITStracking/Vertexer.h

```diff
@@ -7,72 +7,116 @@
 // In applying this license CERN does not waive the privileges and immunities
 // granted to it by virtue of its status as an Intergovernmental Organization
 // or submit itself to any jurisdiction.
+///
+/// \file Vertexer.h
+/// \brief
+///
 
-#ifndef O2_ITSMFT_RECONSTRUCTION_CA_VERTEXER_H_
-#define O2_ITSMFT_RECONSTRUCTION_CA_VERTEXER_H_
+#ifndef VERTEXER_H_
+#define VERTEXER_H_
```

Comment:
What is the reason to go short here? Just `VERTEXER` seems really too short to me and can clash with other headers.

Reply:
Thanks for pointing this out. No reason, updated.